packages/create-sitecore-jss/src/initializers/angular/prompts.ts

```diff
@@ -31,5 +31,6 @@ export const prompts = [
       return true;
     },
   },
+  ...clientAppPrompts,
   ...styleguidePrompts,
```

Comment:
For me seems like we might recieve a prompt "Which additional language do you want to support (en is already included and required)?"
Does it appear? But it's not expected